src/libs/runtime-utils.js

```diff
@@ -37,14 +37,18 @@ export const pdTypes = {
     displayName: 'Solid state drive (SSD)',
     regionToPricesName: 'monthlySSDDiskPrice'
   },
-  fromString: str => Utils.switchCase(str,
-    [pdTypes.standard.label, () => pdTypes.standard],
-    [pdTypes.balanced.label, () => pdTypes.balanced],
-    [pdTypes.ssd.label, () => pdTypes.ssd],
-    [Utils.DEFAULT, () => console.error(`Invalid disk type: Should not be calling pdTypes.fromString for ${str}`)]
-  )
+  fromString: str => {
+    return Utils.switchCase(str,
```

Comment:
you shouldn't need the brackets and return, assuming its leftover from some prints that were put here?